lib/matplotlib/tests/test_collections.py

```diff
@@ -623,7 +621,7 @@ def test_pandas_indexing():
     try:
         import pandas as pd
     except ImportError:
-        raise SkipTest("Pandas not installed")
+        skip("Pandas not installed")
```

Comment:
Actually, now that I've looked into it further, `matplotlib.testing.skip` was added only in `master`, so I think we should definitely not add any more callers if we can. It should just go away once we've fully dropped nose.